BDT_application/step2_BDTapplication_condor/wrapper.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash -e 
 echo "TEST FIRST" 
 echo "copy input root file"
-eos cp /eos/cms/store/group/phys_top/ExtraYukawa/BDT/BDT_input_application/YEAR/SIGNALROOT .
-eos cp /eos/cms/store/group/phys_top/ExtraYukawa/BDT/BDT_input_application/YEAR/aa.tar.gz .
+cp /eos/user/t/tihsu/database/ExtraYukawa_BDT/BDT_input_forApplication//YEAR/SIGNALROOT .
+cp /eos/user/t/tihsu/database/ExtraYukawa_BDT/BDT_input_forApplication//YEAR/aa.tar.gz .
```

Comment:
Hi @AlbertHsuNTUphys You are NOT generating the step1 input file for the application, so why did you change it? 
I would prefer to keep it as it is.

Reply:
Okay, let me restore it.